debian/rules

```diff
@@ -23,6 +24,21 @@ else
 endif
 endif
 
+override_dh_gencontrol:
+ifeq ($(DIST_RELEASE),trusty)
+	echo "extra:Depends=libapt-pkg4.12 (>= 1.0.1ubuntu2.23)" >> debian/ubuntu-advantage-tools.substvars
+else ($(DIST_RELEASE),xenial))
```

Comment:
You need another `ifeq` after `else` in all of these.

Reply:
resolved in a separate commit before this review.